dapps/evancore.vue.libs/src/i18n/en.json

```diff
@@ -67,6 +67,20 @@
     "login": "Unlock profile",
     "logout": "Logout",
     "logout-desc": "Are you sure you want to logout?",
+    "mnemonic-export": {
+      "account-id": "Account ID",
+      "alias": "Alias",
+      "created-at": "Created at:",
+      "description": "Your account is protected by your password and recovery key. You will need your recovery key to log in from new devices.\n\nPlease note: The recovery key is not stored in your profile information and could not be set back. In case of lost, you will loose access to your business partners, contacts and profile information.",
```

Comment:
```suggestion
      "description": "Your account is protected by your password and recovery key. You will need your recovery key to log in from new devices.\n\nPlease note: The recovery key is not stored in your profile information and could not be set back. In case of lost, you will lose access to your business partners, contacts and profile information.",
```